asset-properties-widget/component/asset-property-selector-model/schema-property-selector.component.html

```diff
@@ -0,0 +1,115 @@
+          type="search"
+          class="form-control"
+          placeholder="{{ 'Filter properties' | translate }}"
+          [(ngModel)] ="search"
+        />
+        <span class="input-group-addon" *ngIf="search">
+          <i c8yicon="search" class="dlt-c8y-icon-search" ng-show="search.length === 0 || !search"></i>
+          <i
+            c8yicon="times"
+            class="text-muted dlt-c8y-icon-times"
+            ng-show="search.length > 0"
+            (click) ="search = ''"
+          ></i>
+        </span>
+      </div>
+    </div>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-xs-2">
+          <label>{{ 'Show' | translate }}</label>
+        </div>
+        <div class="col-xs-5">
+          <label>{{ 'Label' | translate }}</label>
+        </div>
+        <div class="col-xs-5">
+          <label>{{ 'Key' | translate }}</label>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div
+    class="modal-inner-scroll container-fluid"
+    id="modal-body"
+    style="max-height: calc(100vh - 340px)"
+  >
+    <div class="row">
+      <div
+        class="d-flex a-i-center property"
+        *ngFor="let property of customProperties | filter: search"
+      >
+        <div class="col-xs-2">
+          <span ng-switch="selectSingle">
+            <label ng-switch-when="true" class="c8y-radio">
+              <span></span>
+            </label>
+            <label ng-switch-default class="c8y-checkbox">
+              <input
+                  type="checkbox"
+                  [(ngModel)]="property.active"
+                  (change)="onSelectProperty(property)"
+                />
+              <span></span>
+            </label>
+          </span>
+        </div>
+        <div
+          class="col-xs-5 text-truncate"
+          title="{{ property.label | translate }}"
+          [ngClass]="{ strong: isComplexProperty(property) }"
+        >
+          <span>
+            <button
+              class="btn btn-dot"
+              type="button"
+              title="{{
+                'Complex property.' | translate
+              }}"
+              *ngIf="isComplexProperty(property)"
+            >
+              <i c8yicon="tags" class="text-info dlt-c8y-icon-tags"></i>
+            </button>
+          </span>
+          {{ property.label?property.label:property.title | translate}}
+        </div>
+        <div class="col-xs-5 text-truncate" title="{{ property.name | translate}}">
+          {{ property.name | translate}}
+        </div>
+      </div>
+      <div class="a-i-center" *ngIf="customProperties.length < 1">
+        {{ 'No property to display' | translate }}
+      </div>
+    </div>
+  </div>
+</div>
+<div class="modal-footer">
+    <button
+      class="btn btn-default"
+      type="button"
+      title="{{ 'Cancel' | translate }}"
+      (click)="onCancelButtonClicked()"
+    >
+      {{ 'Cancel' | translate }}
+    </button>
+    <button
+      class="btn btn-primary"
```

Comment:
add data-cy so that it will be easy for automation, please incorporate in all places of html

Reply:
added data-cy